ha/core/controllers/pcs/node_controller.py

```diff
@@ -170,12 +170,21 @@ def check_cluster_feasibility(self, node_id: str) -> dict:
         """
             Check whether the cluster is going to be offline after node with node_id is stopped.
         Args:
-            node_id (str): Private fqdn define in conf store.
+            node_id (str): Node ID from cluster nodes.
         Returns:
             Dictionary : {"status": "", "msg":""}
         """
-        # Raise exception if node_id is not valid
-        self._is_node_in_cluster(node_id=node_id)
+        # Get the node_name (pvtfqdn) fron nodeid
```

Comment:
```
        nodeid_dict = self._confstore.get(f"{const.PVTFQDN_TO_NODEID_KEY}")
        node_name = ""
        for key, nodeid in nodeid_dict.items():
            if nodeid == node_id:
                node_name = key.split('/')[-1]
        if node_name == "":
            raise ClusterManagerError(f"Failed to get node_name for nodeid: {node_id} from conf_store")
```
repetition of code seen
use method

node_id = get_exact_nodeid(node_id)

Reply:
Added def _get_node_name(self, node_id: str) -> str: